gapic-generator/lib/gapic/presenters/service_presenter.rb

```diff
@@ -251,6 +251,11 @@ def generic_endpoint?
         gem.generic_endpoint?
       end
 
+      # @return [String] The api_version for this service, or empty if not defined.
+      def api_version
+        @service.api_version.nil? ? "" : @service.api_version
```

Comment:
You can replace this with `@service.api_version.to_s` since `nil.to_s == ""`. Same with the call below in ServiceRestPresenter.